sdk/storage/azure-storage-blob/tests/test_container.py

```diff
@@ -1650,6 +1676,41 @@ def test_delete_blobs_with_version_id(self, **kwargs):
         assert response[1].status_code == 202
         assert blob.get_blob_properties().get("version_id") == new_blob_version_id
 
+    @pytest.mark.live_test_only
```

Comment:
Why is this live-only? Feels like `set_custom_default_matcher` was meant to make this recordable? Or did we never get batch working?

Reply:
Hmm, if I had to guess I think we left `set_custom_default_matcher` since we know we wanted to eventually get these recorded (and that was going to be part of the solution), but since the tag is on there I think maybe it wasn't working yet? I'll chop it off and check again 😄